src/modules/healtcheck/healtcheck.controller.ts

```diff
@@ -0,0 +1,76 @@
+import { Controller, Get, Version } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { HealtcheckService } from './healtcheck.service';
+
+@ApiBearerAuth()
+@ApiTags('healtcheck')
+//@Controller('healtcheck')
```

Comment:
Should the file and namings be healthcheck instead of healtcheck?

Reply:
Thanks, changed